lib/cisco_node_utils/cmd_ref/bridge_domain.yaml

```diff
@@ -0,0 +1,44 @@
+# bridge_domain
+# bridge_domain feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k]
+
+all_bds:
+  multiple: true
+  get_command: "show bridge-domain"
```

Comment:
Yes its possilbe and changed it to accommodate it. I have used "show bridge-domain" bridge-domain component is single sourced component accross nxos and its very unlikely that this syntax would be changed. Also another reason was that in show running-config there will be a huge set of configs if its scale setup so thought parsing would take a lot of time.